pom.xml

```diff
@@ -15,7 +15,7 @@
 		<jdk.version>1.8</jdk.version>
 		<!-- When updating one spring version, make sure that all of them are updated to their latest compatible versions -->
 		<spring.boot.version>2.7.4</spring.boot.version>
-		<spring.framework.version>5.3.23</spring.framework.version>
+		<spring.framework.version>5.2.20.RELEASE</spring.framework.version>
```

Comment:
This is the third occurrence of downgrading the Spring Framework jars from the most current release.  Is it not obvious that `5.2.20.Release` is older then `5.3.23`????